### PR TITLE
Update autopilot-faq.yml

### PR DESCRIPTION
**Pull request by mmarable**
When attempting to import a CSV with more than 500 entries, Intune will fail to take the file and state that it is over the 500 line limit. I have a screen shot available if needed.
Thank you.

### CODE REVIEW

File: memdocs/autopilot/autopilot-faq.yml [hunk 1]
```diff
@@ -149,7 +149,7 @@ sections:
       - question: |
           Is there a limit to the number of devices that can be listed in the CSV file?
         answer: |
-          Yes. The CSV file can only contain 1,000 devices to apply to a single profile. If more than 1,000 devices need to be applied to a profile, the devices need to be uploaded through multiple CSV files.
+          Yes. The CSV file can only contain 500 devices to apply to a single profile. If more than 1,000 devices need to be applied to a profile, the devices need to be uploaded through multiple CSV files.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/autopilot-faq.yml:152]:
```suggestion
          Yes. The CSV file can only contain 500 devices to apply to a single profile. If more than 500 devices need to be applied to a profile, the devices need to be uploaded through multiple CSV files.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mmarable, kindly commit the suggested change, Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by frankroj** [comment]
Hello @mmarable. Can you please provide the screenshot mentioned. I need to confirm if this is expected behavior or a possible issue. Thank you.

**Comment by mmarable** [comment]
Certainly.  It is attached.

I tried importing a CSV with ~800 lines and received the message pictured.

Thanks
Mike
________________________________

**Comment by frankroj** [comment]
@mmarable can you please reply with screenshot directly at github.com? It does not look like the screenshot is making it through via email. Thanks.

**Comment by mmarable** [comment]
Sure. I’ll see if I can get it uploaded there.

Mike


Sent from my iPhone

On Feb 10, 2023, at 5:24 PM, Frank Rojas ***@***.***> wrote:

﻿

@mmarable<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmmarable&data=05%7C01%7C%7C379dac4b52dc4c73d23f08db0bb58b16%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638116646577989450%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=9QdxPtJv3xPB1KUMZrQVn3zNsFVjXLjZWALS9e4SaKY%3D&reserved=0> can you please reply with screenshot directly at github.com? It does not look like the screenshot is making it through via email. Thanks.

**Comment by mmarable** [comment]
> Sure. I’ll see if I can get it uploaded there. Mike
> […](#)

D6D8106E-7FA8-4BAE-ABDE-8399BF187451

**Pull request merged**